lightning/src/chain/keysinterface.rs

```diff
@@ -341,12 +341,10 @@ pub trait KeysInterface: Send + Sync {
 	/// Get a new set of ChannelKeys for per-channel secrets. These MUST be unique even if you
 	/// restarted with some stale data!
 	fn get_channel_keys(&self, inbound: bool, channel_value_satoshis: u64) -> Self::ChanKeySigner;
-	/// Get a secret and PRNG seed for constructing an onion packet
-	fn get_onion_rand(&self) -> (SecretKey, [u8; 32]);
-	/// Get a unique temporary channel id. Channels will be referred to by this until the funding
-	/// transaction is created, at which point they will use the outpoint in the funding
-	/// transaction.
-	fn get_channel_id(&self) -> [u8; 32];
+	/// Gets a unique, cryptographically-secure, random 32 byte value. This is used for encrypting
```

Comment:
Is this an API requirement change to switch from using a PRNG to CSPRNG ? This is better, but if so rename `prng_seed` mentions.

Reply:
I only see one doc comment mentioning RNG.